messages/views.py

```diff
@@ -107,28 +110,33 @@ def message(request, message_id):
     tvars = {'message': message}
     return render(request, 'messages/message.html', tvars)
 
+
 @login_required
 @transaction.atomic()
 def new_message(request, username=None, message_id=None):
+
+    if request.user.profile.num_sounds > 0 or request.user.profile.num_posts > 5:
+        form_class = MessageReplyForm
+    else:
+        form_class = MessageReplyFormWithCaptcha
     
     if request.method == 'POST':
-
-        if request.user.profile.num_sounds:
-            form = MessageReplyFormNoCaptcha(request.POST)
-        else:
-            form = MessageReplyForm(request.POST)
+        form = form_class(request.POST)
 
         if request.user.profile.is_blocked_for_spam_reports():
-            messages.add_message(request, messages.INFO, "You're not allowed to send the message because your account has been temporaly blocked after multiple spam reports")
+            messages.add_message(request, messages.INFO, "You're not allowed to send the message because your account "
+                                                         "has been temporaly blocked after multiple spam reports")
         else:
             if form.is_valid():
                 user_from = request.user
                 user_to = form.cleaned_data["to"]
                 subject = form.cleaned_data["subject"]
                 body = MessageBody.objects.create(body=form.cleaned_data["body"])
 
-                Message.objects.create(user_from=user_from, user_to=user_to, subject=subject, body=body, is_sent=True, is_archived=False, is_read=False)
-                Message.objects.create(user_from=user_from, user_to=user_to, subject=subject, body=body, is_sent=False, is_archived=False, is_read=False)
+                Message.objects.create(user_from=user_from, user_to=user_to, subject=subject, body=body, is_sent=True,
```

Comment:
I wonder if we should do something about the `is_sent` field. It always confuses me when I see it. I expect it to mean that the message has been successfully sent to someone, rather than indicating that this is the message in the output of the sender. Any suggestions on what else it could be called?

Reply:
that's a tricky one. maybe `is_sender_copy`?

Reply:
let's leave it like this for now, but consider changing it if we make any further changes to messages